packages/react-composites/src/composites/CallComposite/CallComposite.tsx

```diff
@@ -329,34 +329,12 @@ export const CallComposite = (props: CallCompositeProps): JSX.Element => {
         adapter.querySpeakers();
         return;
       }
-      /* @conditional-compile-remove(call-readiness) */
```

Comment:
Looks good.
Thanks for cleaning this up. Now with the buttons, it should be good to remove this.